src/components/CommentUsername/CommentUsername.js

```diff
@@ -33,15 +35,15 @@ const CommentUsername = ({comment, user, isPreviewComment}) => {
     return (
       <Text style={[styles.username, styles.previewContainer]}>
         {comment.data?.anon_user_info_color_name
-          ? `Anonymous ${comment.data?.anon_user_info_emoji_name}`
+          ? getOfficialAnonUsername(comment?.data)
           : user?.data?.username}{' '}
       </Text>
     );
   }
   return (
     <Text style={styles.username}>
       {comment.data?.anon_user_info_color_name
-        ? `Anonymous ${comment.data?.anon_user_info_emoji_name}`
+        ? getOfficialAnonUsername(comment?.data)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the above comment, ensure that `comment?.data` exists before checking for `comment.data.anon_user_info_color_name` and passing `comment?.data` to `getOfficialAnonUsername`.

```diff
- {comment.data?.anon_user_info_color_name ? getOfficialAnonUsername(comment?.data) : user?.data?.username}{' '}
+ {comment?.data && comment.data.anon_user_info_color_name ? getOfficialAnonUsername(comment.data) : user?.data?.username}{' '}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->